core/components/molecules/pager/pager.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Button from '../../atoms/button'
+
+import { changePageIfAppropiate, totals } from '../../_helpers/pagination'
+
+const StyledPager = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const StyledPageSelector = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`
+
+const Pager = ({ onPageChanged, page, perPage, items }) => (
+  <StyledPager>
+    <Button
+      size="small"
+      appearance="secondary"
+      onClick={() => changePageIfAppropiate(page - 1, items, perPage, onPageChanged)}
+    >
+      {'<'} Newer
```

Comment:
We should replace the arrows with icons. You can achieve this with `chevron-down` + rotation if you feel like it

@landitus request for `chevron-left` + `chevron-right`